Note about capturing freertos variables (#553)
memfault/memfault-docs: docs/mcu/rtos-analysis.mdx

  extern uint32_t __memfault_capture_bss_start;
  extern uint32_t __memfault_capture_bss_end;
  const size_t memfault_region_size = (uint32_t)&__memfault_capture_bss_end -
      (uint32_t)&__memfault_capture_bss_start;

  s_coredump_regions[region_idx] = MEMFAULT_COREDUMP_MEMORY_REGION_INIT(
    &__memfault_capture_bss_start, memfault_region_size);
  region_idx++;

  // remaining regions would go here. typically the
  // 'memfault_freertos_get_task_regions()' helper would be used to capture the
  // task data.
  region_idx += memfault_freertos_get_task_regions(&s_coredump_regions[region_idx],
      MEMFAULT_ARRAY_SIZE(s_coredump_regions) - region_idx);
  // remaining regions
```

  </TabItem>

  <TabItem value="IAR">

Update the linker script (typically a file with a `.icf` extension) to include
this line:

```linker-script
define block memfault_freertos { section .bss* object *tasks*.o*, section .bss* object *timers*.o* };
```

And then update `memfault_platform_coredump_get_regions()` to include that
block:

```c
// In the source file where memfault_platform_coredump_get_regions is defined add the following snippet

> // Be sure to include an additional sMfltCoredumpRegion in your regions array for the FreeRTOS region
> // created by the linker script

Reviewer: ```suggestion
// Be sure to include additional sMfltCoredumpRegion entries in your regions array for the FreeRTOS regions
// created by the linker script and helper function
```
